app/src/main/java/com/example/bigowlapp/model/Group.java

```diff
@@ -55,4 +54,13 @@ public void setMemberIdList(List<String> memberIdList) {
         this.memberIdList = memberIdList;
     }
 
+    public static class Field {
+        public static final String NAME = "name";
+        public static final String SUPERVISOR_ID = "supervisorId";
+        public static final String MEMBER_ID_LIST = "memberIdList";
+
+        private Field() {
+            // constants class should not be instantiated
+        }
+    }
```

Comment:
Very good way to store constants.

I prefer this way than keeping the constants in an outer class.